appendix/configure-database-server.rst

```diff
@@ -27,7 +27,7 @@ Keep in mind that versions may differ from your setup - adapt where needed.
 
       .. code-block::
 
-         /etc/postgresql/(10|11|12)/main/postgresql.conf
+         /etc/postgresql/(10|11|12|13|14|15)/main/postgresql.conf
```

Comment:
Looks good, but we will have to keep updating this one every few months. What about `/*/` or similar instead?

Reply:
You are right. I suggest something like /etc/postgresql/{your version}/main/postgresql.conf